packages/core/ios/EmbraceManager.swift

```diff
@@ -808,4 +832,9 @@ class EmbraceManager: NSObject {
 
         resolve(true)
     }
+
+    func injectW3cTraceparent(span: OpenTelemetryApi.Span) {
```

Comment:
I think this would work without fully qualifying it:
```suggestion
    func injectW3cTraceparent(span: Span) {
```